src/utils.ts

```diff
@@ -1,18 +1,71 @@
-import { IUser } from "./models/user";
+import axios from "axios";
+import dotenv from "dotenv";
+
 import { Note } from "./models/note";
+import { Request, Response } from "express";
+
+dotenv.config();
 
-export const createMongoNoteInstance = async (
-  sender: IUser,
-  receiver: IUser,
-  status: string,
-  url: string
+export const saveNoteToDB = async (
+  slackID: string,
+  audioUrl: string,
+  responseUrl: string
 ) => {
-  if (sender._id != null && receiver._id != null) {
+  try {
     return new Note({
-      sender: sender._id,
-      receiver: receiver._id,
-      status,
-      url,
+      slackID,
+      audioUrl,
+      responseUrl,
     }).save();
+  } catch (error) {
+    console.log(error);
+  }
+};
+
+const returnUrlParameters = (url: string) => {
+  const splitUrl = url.split("/");
+  const index = splitUrl.length;
+  return [splitUrl[index - 3], splitUrl[index - 2], splitUrl[index - 1]];
+};
+
+export const slackQuery = (request: Request, response: Response) => {
+  const { user_id, response_url } = request.body;
+  if (response_url) {
+    const [param1, param2, param3] = returnUrlParameters(response_url);
+
+    response.send({
+      blocks: [
+        {
+          type: "section",
+          text: {
+            type: "mrkdwn",
+            text: `<http://localhost:3000/?&sender=${user_id}&p1=${param1}&p2=${param2}&p3=${param3}|Tape your message here!>`,
+          },
+        },
+      ],
+    });
+  }
+};
+
+export const postToSlackWebhook = async (
+  audioUrl: string,
+  responseUrl: string
+) => {
+  try {
+    let res = await axios.post(`${responseUrl}`, {
```

Comment:
Why is this a let?

Why did you decide to use axios over other things, like for example fetch?

Reply:
Have changed the let to a const

Just went with axios because the node.js docs suggested it was the simplest method (https://nodejs.dev/learn/make-an-http-post-request-using-nodejs)

Happy to change to fetch if you'd prefer less abstraction?